lib/ts/nextjs.ts

```diff
@@ -39,6 +39,7 @@ export default class NextJS {
         response: any
     ): Promise<T> {
         return new Promise<T>(async (resolve: any, reject: any) => {
+            request.fromNextJS = true;
```

Comment:
I prefer a name like `request.__supertokensFromNextJS` just so it's clear this is a var from us